apollo-compiler/src/main/kotlin/com/apollostack/compiler/ir/GraphQlType.kt

```diff
@@ -1,11 +1,13 @@
 package com.apollostack.compiler.ir
 
+import com.apollostack.compiler.normalizeTypeName
+import com.squareup.javapoet.AnnotationSpec
 import com.squareup.javapoet.ClassName
 import com.squareup.javapoet.ParameterizedTypeName
 import com.squareup.javapoet.TypeName
+import javax.annotation.Nullable
```

Comment:
android support annotations is neither on jcenter or maven central, so we're gonna have to stick to jsr305